docs/sources/whatsnew/whats-new-in-v9-5.md

```diff
@@ -0,0 +1,180 @@
+
+_Experimental in all editions of Grafana._
+
+New feature toggle `prometheusResourceBrowserCache` provides the ability to cache Prometheus editor API calls in the Prometheus data source configuration.
+This improves Prometheus query editor performance, with the biggest performance improvements seen by users with high cardinality Prometheus instances.
+
+## Removal of API key creation from the UI
+
+With this update we are going one step further in deprecating API Keys in favor of [service accounts]({{< relref "../administration/service-accounts/" >}}). We've removed the button for creating new API keys through Grafana's user interface, and now only allow the creation of API keys using our HTTP API. We recommend that you migrate your existing API keys to service accounts, and opt for new service accounts instead of new API keys. This change is part of our long-term strategy for sunsetting API keys.
+
+Learn more about the deprecation strategy for API keys and how to manage them in our [Sunsetting API keys](https://github.com/grafana/grafana/issues/53567) GitHub issue.
+
+## Resolve Grafana issues faster with support bundles
+
+_Generally available in all editions of Grafana._
+
+Support bundles provide a simple way to collect information about your Grafana instance through Grafana's user interface. In a few clicks, you can create a support bundle containing data about migrations, plugins, settings, and more. Once you've created a support bundle, you can either examine it yourself, or share it with your colleagues or Grafana engineers to aid in troubleshooting of your Grafana instance.
+
+Learn more about support bundles and how to configure them in our [support bundle documentation]({{< relref "../troubleshooting/support-bundles/" >}}).
+
+{{< figure src="/static/img/docs/troubleshooting/support-bundle.png" max-width="750px" caption="Create a support bundle to resolve issues faster" >}}
+
+## Alerting
+
+_All Alerting improvements are generally available in all editions of Grafana._
+
+### Search for alert rules from multiple data sources
+
+Search for and display alert rules for multiple data sources at the same time.
+
+### Fuzzy search on the Alert rule list view
+
+Search for namespaces or folders, evaluation groups, and alert rule names on the Alert rules list view with immediate results, and regardless of typos.
+
+### Access an alert rule from a dashboard or a panel
+
+Navigate to an alert rule directly from a dashboard or a panel to easily access the alert rule details.
+
+{{< figure src="/media/docs/alerting/alert-rule-dashboard.png" max-width="750px" caption="Access an alert rule from a dashboard or a panel" >}}
+
+### Access a dashboard or panel from an alert rule
+
+Navigate from an alert rule straight to a dashboard or a panel associated with the alert rule to visualize your alerting data.
+
+{{< figure src="/media/docs/alerting/dashboard-alert-rule.png" max-width="750px" caption="Access a dashboard or panel from an alert rule" >}}
```

Comment:
@brendamuir - Is it possible to get this screenshot in dark mode to be consistent with all the other ones in the What's New?

Reply:
hi there-i've asked for it. most people are off today-hope to have it by tomorrow!

Reply:
Added!

Reply:
thank you!